src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -1022,6 +1017,43 @@ public void saveHeartbeat() {
     }
   }
 
+
+  private static void overrideQueryOptions(CassandraFactory cassandraFactory) {
+    // all INSERT and DELETE stmt prepared in this class are idempoten
+    if (cassandraFactory.getQueryOptions().isPresent()
+        && ConsistencyLevel.LOCAL_ONE != cassandraFactory.getQueryOptions().get().getConsistencyLevel()) {
+      LOG.warn("Customization of cassandra's queryOptions is not supported and will be overridden");
+    }
+    cassandraFactory.setQueryOptions(java.util.Optional.of(new QueryOptions().setDefaultIdempotence(true)));
+  }
+
+  private static void overrideRetryPolicy(CassandraFactory cassandraFactory) {
+    if (cassandraFactory.getRetryPolicy().isPresent()) {
+      LOG.warn("Customization of cassandra's retry policy is not supported and will be overridden");
+    }
+    cassandraFactory.setRetryPolicy(java.util.Optional.of((RetryPolicyFactory) () -> new RetryPolicyImpl()));
+  }
+
+  private static void overridePoolingOptions(CassandraFactory cassandraFactory) {
+    PoolingOptionsFactory newPoolingOptionsFactory = new PoolingOptionsFactory() {
+      @Override
+      public PoolingOptions build() {
+        if (null == getPoolTimeout()) {
+          setPoolTimeout(Duration.seconds(20));
+        }
+        return super.build().setMaxQueueSize(4096);
```

Comment:
i'd like to bump this up to 4k. writes in-flight are in-memory and non-durable. 
so the longer we can hold them for: the more durable they are.